src/Response/Struct/ShipmentAddress.php

```diff
@@ -47,7 +47,6 @@ protected function setBaseShipmentProperties(array $addressData): void
         $this->streetId = $addressData['streetId'] ?? null;
         $this->streetType = $addressData['streetType'] ?? null;
         $this->streetName = $addressData['streetName'] ?? null;
-        $this->streetType = $addressData['streetType'] ?? null;
```

Comment:
why was this removed?

Reply:
duplicated, 2 lines above is the the same seter